plugins/adr/src/search/AdrSearchResultListItem.tsx

```diff
@@ -104,6 +106,13 @@ export const AdrSearchResultListItem = ({
           }
         />
         <Box>
+          <Chip
+            label={`Entity: ${
+              result.entityTitle ??
+              humanizeEntityRef(parseEntityRef(result.entityRef))
+            }`}
+            size="small"
```

Comment:
Don't you want to make this one clickable, going to the entity page? Maybe a future addition.

Reply:
hm it seemed a bit redundant since click on the search result itself would lead to the entity page (via it's ADR plugin tab)